wallets/provider-math-wallet/src/index.ts

```diff
@@ -83,6 +87,7 @@ export const getWalletInfo: (allBlockChains: BlockchainMeta[]) => WalletInfo = (
 ) => {
   const evms = evmBlockchains(allBlockChains);
   const solana = solanaBlockchain(allBlockChains);
+  const cosmos = cosmosBlockchains(allBlockChains);
```

Comment:
If Math Wallet is designed specifically for the Cosmos blockchain, it's crucial to return only the meta for the Cosmos blockchain instead of including all blockchains with the transaction type as Cosmos. This is essential to avoid potential issues in other packages like widget-embedded.

Reply:
Fixed. Thanks